### PR TITLE
updating docs with additional prereqs and setup playbook

### PR DESCRIPTION
**Pull request by kkbankol-ibm**
Adding additional prerequisites for the nova python client, and adding a call to the setup.yml playbook to generate the edge ssl certs.

### CODE REVIEW

**Comment by markusthoemmes** [comment]
What's the status of this @kkbankol-ibm ?

**Comment by kkbankol** [comment]
Hey @markusthoemmes, this should be ready to be merged in unless you see any issues with it

**Comment by kkbankol-ibm** [comment]
Hey @markusthoemmes, this should be ready to be merged in unless you see any issues with it

File: ansible/README_DISTRIBUTED.md [hunk 1]
```diff
@@ -15,10 +15,8 @@ Login to your bootstrapper VM. Your local machine can act as the bootstrapping m
 This installs modules and packages to manage cloud instances via Ansible.
 
 ```
-sudo apt-get -y install python-setuptools python-dev libssl-dev
-sudo pip install shade pytz positional appdirs monotonic rfc3986
-sudo apt-get install python-novaclient
-sudo pip install six --upgrade
+sudo apt-get -y install python-setuptools python-dev libssl-dev build-essential libssl-dev libffi-dev python-dev python-novaclient
+sudo pip install shade pytz positional appdirs monotonic rfc3986 pyparsing stevedore debtcollector netaddr oslo.config futures warlock six
```
Comment (ddragosd) [inline ansible/README_DISTRIBUTED.md]: One idea to explore to make it easier is to try to move these commands into the `setup.yaml` or install them from a `dependencies.txt` file as: `pip install -r dependencies.txt`, also controlling the versions.

**Pull request merged**